pom.xml

```diff
@@ -332,25 +331,25 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-assembly-plugin</artifactId>
-        <version>2.2</version>
+        <version>3.1.0</version>
       </plugin>
 
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-jar-plugin</artifactId>
-        <version>2.3.1</version>
+        <version>3.1.1</version>
       </plugin>
 
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-install-plugin</artifactId>
-        <version>2.3.1</version>
+        <version>3.0.0-M1</version>
```

Comment:
Do you happen to know if the `-M1` means it's a pre-release milestone or something similar? It was released in September so it's probably fine. We could consider sticking to the last `2.x` release which was `2.5.2`, but that was nearly 5 years ago...

https://search.maven.org/search?q=g:org.apache.maven.plugins%20AND%20a:maven-install-plugin&core=gav

Reply:
I'm using the `-M1` version since last year, and it's working fine.